drivers/usbhid-ups.c

```diff
@@ -1348,6 +1349,21 @@ void upsdrv_initups(void)
 		}
 	}
 
+	val = getval("onlinedischarge_log_throttle_hovercharge");
```

Comment:
One point cursorily raised in another discussion (#2430) and posted to mailing list is that *maybe* this setting name and internal NUT concept could boil down to a `(default.|override.)battery.charge.high` (not currently defined in `docs/nut-names.txt`) and/or a `HB` flag. At least, if we as a community remember and define what such a flag means - some alert about battery overcharge as harmful for its health, or something similar/identical to hovering over a "well-charged" state which is not 100% (max voltage) exactly but e.g. fluctuating between 85-95-100%. The downwards part of this battery state cycle, depleting from ~100% to the "high" watermark, would in fact be "online and discharging".

Probably not gonna be a focus of *this* PR, but can well pop up as continuation of the subject.

Reply:
Probably hard to set up for the end-user without knowing what voltage a UPS actually considers as 100% and at what percentage (compared to voltage) it is in fact starting to be "harmful" for the battery, especially when we're dealing with percentages rather than voltages. And even if the end-user were alerted of reaching such a threshold "high" percentage - what would be the consequence? We can't stop charging the battery after all, a shutdown probably also is out of the question, not sure what the benefit could be other than knowing. If I understood the idea correctly there... 😎